matrix-editor/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "matrix-editor",
+  "version": "1.0.0",
+  "main": "lib/script.js",
+  "repository": "https://github.com/chchch/sanskrit-alignment",
+  "author": "Charles Li",
+  "license": "BSD-3-Clause",
+  "type": "module",
+  "private": true,
+  "files": [
+    "lib"
+  ],
+  "scripts": {
+    "build": "webpack",
+    "test": "mocha 'test/**/*.spec.+(js|jsx)'"
+  },
+  "dependencies": {
+    "file-saver": "^2.0.5",
+    "raphael": "^2.3.0"
```

Comment:
Dependencies would go here :point_up: instead of copied into the `lib` directory. It is also possible to install from GitHub, ex: https://stackoverflow.com/a/17509764